action.yaml

```diff
@@ -49,6 +49,11 @@ inputs:
     description: Maximum number of iterations to resolve dependencies.
     required: false
     default: 10
+  repository-list:
+    description: |
+      Comma-separated list of pkgcache-compatible repository names from which the dependencies will be downloaded.
+    required: false
+    default: "https://pharmaverse.r-universe.dev,PPM@latest"
```

Comment:
```suggestion
    default: "PPM@latest"
```

Reply:
I think I agree. If runiverse is required then we can specify it when running the action. The default should be just ppm